lib/events/events.proto

```diff
@@ -1093,6 +1093,9 @@ message AppSessionRequest {
     string Path = 3 [ (gogoproto.jsontag) = "path" ];
     // RawQuery are the encoded query values.
     string RawQuery = 4 [ (gogoproto.jsontag) = "raw_query" ];
+    // Method defines the action to be performed on the resource, like GET,
```

Comment:
Nit: I would just say `Method is the request HTTP method, like GET/POST/DELETE/etc.` to use standard terminology.